apigentools/cli.py

```diff
@@ -20,6 +20,12 @@ def get_cli_parser():
     p = argparse.ArgumentParser(
         description="Manipulate OpenAPI specs and generate code using openapi-generator",
     )
+    p.add_argument(
+        '--git-via-https',
+        action='store_true',
+        default=env_or_val('APIGENTOOLS_GIT_VIA_HTTPS', False),
```

Comment:
You should use `_type=bool` as argument for `env_or_val`, see documentation of this function.